### PR TITLE
mondialerelayformulaire.com

### PR DESCRIPTION
**Pull request by Totorh**
Phishing site for debit cards in Belgium, copying mondialrelay.be

### CODE REVIEW

File: add-domain [hunk 1]
```diff
@@ -522,3 +522,4 @@ pdfkinghub.xyz
 tracebb.trackingsystem.xyz
 postgo.webpostgo.icu
 totaltrkonups2298.fitritomoastramencostea.com
```
Comment (spirillen) [inline add-domain:524]: Can only see a default Apache Centos page. Do you have any other information for confirmation?

**Review by spirillen — CHANGES_REQUESTED** [review]
Not for change, but need more info for confirmation

**Comment by iam-py-test** [comment]
@Totorh any update?

**Comment by spirillen** [comment]
Closed as unresponsive

**Pull request closed without merging**